partcad/src/partcad/project.py

```
@@ -1426,6 +1427,60 @@ def render(
+                if hasattr(shape, f"render_{target_format}_async")
+            ]
+            await asyncio.gather(*tasks)
+
+            if in_place:
+                for shape in shapes:
+                    self.update_part_config(shape.name, {"type": target_format})
+
+
+    def convert(
+        self,
+        sketches: Optional[List[str]] = None,
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)